### PR TITLE
net-nntp/sabnzbd: add 4.3.2

### PR DESCRIPTION
**Pull request by joecool1029**
It's fine to use the newer sabctools that's already in ::gentoo. Tests pass, it works, I think they just forgot to bump the dependency before making the release. 

---

Please check all the boxes that apply:

- [X] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [X] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [X] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [X] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

**Comment by joecool1029** [comment]
And now tests are breaking.... https://bugs.gentoo.org/934331

**Comment by joecool1029** [comment]
Disabling the failing test for this release, it's fixed for 4.3.3 but that's still some weeks out. Otherwise, this is ready now.

File: net-nntp/sabnzbd/sabnzbd-4.3.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+	test? (
+		$(python_gen_cond_dep '
+			dev-python/flaky[${PYTHON_USEDEP}]
+			>=dev-python/lxml-4.5.0[${PYTHON_USEDEP}]
+			<dev-python/pyfakefs-5.4.0[${PYTHON_USEDEP}]
+			dev-python/pytest-asyncio[${PYTHON_USEDEP}]
+			dev-python/pytest-httpbin[${PYTHON_USEDEP}]
+			dev-python/pytest-httpserver[${PYTHON_USEDEP}]
+			dev-python/pytest-mock[${PYTHON_USEDEP}]
+			~dev-python/pytest-7.4.4[${PYTHON_USEDEP}]
```
Comment (juippis) [inline net-nntp/sabnzbd/sabnzbd-4.3.2.ebuild:67]: Is this still true? For some reason this restriction makes it ... very hard if not impossible to test this package for some reason.

Comment (juippis) [reply net-nntp/sabnzbd/sabnzbd-4.3.2.ebuild:67]: Tests passed once I got all the deps installed so overall package seems fine. I'll wait for your confirmation on this though.

Comment (joecool1029) [reply net-nntp/sabnzbd/sabnzbd-4.3.2.ebuild:67]: yeah it's still true, I can try newer if you'd like but upstream has a limit on the version.

**Pull request closed without merging**